examples/basic/main.tf

```diff
@@ -2,46 +2,41 @@ provider "azurerm" {
   features {}
 }
 
-resource "azurerm_resource_group" "example" {
-  name     = "example-resources"
-  location = "West Europe"
+resource "azurerm_resource_group" "rg-connectivity-01" {

```

Comment:
Conventionally in terraform resources are named using snake_case for example "rg_connectivity_01" or I would prefer just "connectivity"